### DISCUSSION TITLE
How to use the /engines/acp/ory/exact/allowed endpoint

### DISCUSSION THREAD

**Discussion by kaiba42** [ory/keto#1415] [Q&A]
I'm trying to use Oathkeeper and Keto to authorize requests to my API.
I see that [Oathkeeper calls Keto's endpoint](https://github.com/ory/oathkeeper/blob/1329413f1ec997daaa15a94e94342c54114383f2/pipeline/authz/keto_engine_acp_ory.go#L153)  `/engines/acp/ory/exact/allowed endpoint`. 
I cannot find any documentation on this endpoint. Where is this endpoint defined in the source code? What is the API? What are the inputs and outputs?

**Comment by vinckr** [comment]
Hello @kaiba42 
This is what I know: 
There is an authorizer named `keto_engine_acp_ory` in Ory Oathkeeper that uses the Ory Keto API to carry out access control using "Ory-flavored" Access Control Policies. The configuration for this authorizer includes base_url, required_action, required_resource, subject, and flavor parameters. You can find more details about this in the [Ory documentation](https://www.ory.sh/docs/oathkeeper/pipeline/authz).

>Where is this endpoint defined in the source code?

I am not sure, but you can take a look yourself 😉

**Comment by rrmistry** [comment]
After some digging found out the status for `keto_engine_acp_ory` authorizer.

Linking some of these discussions together:
https://community.ory.sh/t/problems-with-configuration-of-keto-engine-acp-ory-authorizer/2079/2

Summary:

> We want to deprecate `keto_engine_acp_ory` in favor of a generic `remote_json` handler which is why we haven’t shown it a lot of love recently. The problem is that `keto_engine_acp_ory` currently only works with regex routes if I recall correctly, because of the way we use RegexCaptureGroups to render `required_action` and so on. Hope this helps!

Also mentioned in: https://github.com/ory/oathkeeper/issues/945
